RELEASES.md

```diff
@@ -0,0 +1,23 @@
+# IBM-ROKS Toolkit
+
+## Overview
+For every PR that is merged into this repository, a post-commit CI job checks whether a release should be created. If so,
+it will create a new release by publishing release artifacts to the git repository and tagging images in the CI registry
+with the release tag.
+
+Releases are controlled by 2 files in the root of the repository:
+
+- release - Contains the major.minor version of the ibm-roks-toolkit branch. This should remain constant and only change
+  when a new release branch is created.
+- release-date - The release date in yyyymmdd format. This is used to create the release tag which is in the format:
+  `v[RELEASE]-[RELEASE_DATE]`
```

Comment:
nit: `v[RELEASE]-[RELEASE_DATE]` should be `v[RELEASE]+[RELEASE_DATE]`

Reply:
Thanks! https://github.com/openshift/ibm-roks-toolkit/pull/398